tests/MySqlShimTest.php

```diff
@@ -930,7 +930,8 @@ public function mysql_fetch_DataProvider()
         return [
             [
                 'function' => 'mysql_fetch_array',
-                'results' => $array
+                'results' => $array,
+                'resulttype' => 3
```

Comment:
This can use the standard `MYSQL_BOTH` constant, and there should be more tests added for the other types (`MYSQL_NUM` and `MYSQL_ASSOC`)

Do you want to go ahead and make those changes?